### ISSUE TITLE
Insufficient funds returned by call to `exportAVA` 

### ISSUE THREAD

**Issue by danlaine** [ava-labs/gecko#123]
**Describe the bug**
User gets `insufficient funds` error message when calling `avm.exportAVA` even though their user controls an account with sufficient balance.

**To Reproduce**
User does: 
```sh
curl -X POST --data '{ "jsonrpc":"2.0", "id" :1, "method" :"avm.exportAVA", "params" :{ "to":"Nsscj17QRsQJjppjnGZ9BF5qbMNiGKeTN", "amount": 10000, "username":"xx", "password":"xxx" } }' -H 'content-type:application/json;' 127.0.0.1:9650/ext/bc/X
```

and gets:
```
{"jsonrpc":"2.0","error":{"code":-32000,"message":"insufficient funds","data":null},"id":1}
```

A call to `getBalance` confirms their balance is 40,000 (ie >10,000)

**Expected behavior**
API call succeeds

**Operating System**
Unknown

**Additional context**
Thanks to Discord user Okesip for reporting 

By submitting this issue I agree to the [Terms and Conditions](https://files.avalabs.org/legal/bug_hunt_q2_2020_terms.pdf) of the [Developer Accelerator Program](https://avalabs.org/dap).

**Comment by ilhamm24** [comment]
Not just `avm.exportAVA`, it happens if i send tokens using `avm.send`.

`{"jsonrpc":"2.0","error":{"code":-32000,"message":"insufficient funds","data":null},"id":5}`

Discord : @OkeSip #7763

**Comment by danlaine** [comment]
Probably resolved by #132
Closing for now, will re-open if more people have this issue.
